src/reaper_osara.cpp

```diff
@@ -402,16 +402,41 @@ string formatCursorPosition(TimeFormat format, FormatTimeCacheRequest cache) {
 	return formatTime(GetCursorPosition(), format, false, cache);
 }
 
-const char* formatFolderState(int state, bool reportTrack=true) {
+const string formatFolderState(MediaTrack* track) {
```

Comment:
There's no benefit in this being const now. Previously, it was const because this returned a constant defined at compile time, so we didn't want it to get modified. Now, the string is always generated at runtime, so it must always be a different string and thus there's no reason to prevent the caller from modifying it.